File: src/targets/ruby/native/fixtures/nested.rb
require 'uri'
require 'net/http'
require 'openssl'

<<<<<<< HEAD
url = URI("https://httpbin.org/anything?foo%5Bbar%5D=baz%2Czap&fiz=buz&key=value")
=======
url = URI("https://httpbin.org/anything")
>>>>>>> upstream/master

http = Net::HTTP.new(url.host, url.port)
http.use_ssl = true

request = Net::HTTP::Get.new(url)

response = http.request(request)
puts response.read_body

File: src/targets/node/native/fixtures/nested.js
<<<<<<< HEAD
const http = require('https');

const options = {
  method: 'GET',
  hostname: 'httpbin.org',
  port: null,
  path: '/anything?foo%5Bbar%5D=baz%2Czap&fiz=buz&key=value',
=======
const http = require('http');

const options = {
  method: 'GET',
  hostname: 'mockbin.com',
  port: null,
  path: '/har?foo%5Bbar%5D=baz%2Czap&fiz=buz&key=value',
>>>>>>> upstream/master
  headers: {}
};

const req = http.request(options, function (res) {
  const chunks = [];

  res.on('data', function (chunk) {
    chunks.push(chunk);
  });

  res.on('end', function () {
    const body = Buffer.concat(chunks);
    console.log(body.toString());
  });
});

req.end();